### PR TITLE
Modular code for table structure and table related files

### PR DESCRIPTION
**Pull request by Piyush3079**
This pr contains the modular code for table-related files.

This pr is in continuation with the pr Mod_Js_Db_Structure. Before reviewing this pr, review and merge Mod_Js_Db_Structure and then start reviewing this.

Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Comment by devenbansod** [comment]
@Piyush3079 please rebase

File: js/functions.js [hunk 1]
```diff
@@ -87,127 +87,127 @@ $.ajaxPrefilter(function (options, originalOptions, jqXHR) {
...
-            // for TIME and DATE data-type
-            if ($(this).hasClass('timefield')) {
-                PMA_tooltip($(this), 'input', PMA_messages.strMysqlAllowedValuesTipTime);
-            } else if ($(this).hasClass('datefield')) {
-                PMA_tooltip($(this), 'input', PMA_messages.strMysqlAllowedValuesTipDate);
-            }
-        });
-    }
-}
+// /*
```
Comment (devenbansod) [inline js/functions.js]: /minor remove if not needed?

File: js/functions.js [hunk 2]
```diff
@@ -999,93 +999,93 @@ AJAX.registerOnload('functions.js', function () {
...
-
-    /**
-     * Add attribute to text boxes for iOS devices (based on bugID: 3508912)
-     */
-    if (navigator.userAgent.match(/(iphone|ipod|ipad)/i)) {
-        $('input[type=text]').attr('autocapitalize', 'off').attr('autocorrect', 'off');
-    }
-});
+// /**
+//  * Unbind all event handlers before tearing down a page
```
Comment (devenbansod) [inline js/functions.js]: same as above

File: js/src/ajax.js [hunk 3]
```diff
@@ -772,6 +772,40 @@ export let AJAX = {
 $(document).on('click', 'a', AJAX.requestHandler);
 $(document).on('submit', 'form', AJAX.requestHandler);
 
+/**
+ * @todo this is to be removed when complete code is modularised	 * Gracefully handle fatal server errors
```
Comment (devenbansod) [inline js/src/ajax.js]: 👍

File: libraries/classes/Controllers/Table/TableSearchController.php [hunk 4]
```diff
@@ -244,16 +243,15 @@ public function indexAction()
                 ->getScripts()
                 ->addFiles(
                     [
-                        'makegrid.js',
-                        'sql.js',
+                        'sql',
                         'vendor/jqplot/jquery.jqplot.js',
```
Comment (devenbansod) [inline libraries/classes/Controllers/Table/TableSearchController.php]: Are these still needed to be added explicitly?

**Review by devenbansod — APPROVED** [review]
Tested locally. LGTM.

A few minor comments.

File: libraries/classes/Controllers/Table/TableSearchController.php (hunk 4 above)
Comment (Piyush3079) [reply libraries/classes/Controllers/Table/TableSearchController.php]: Yeah they were sending the scripts list in ajax response but scripts were loaded by dynamic imports only.

**Pull request merged**